lms/models/course_groups_exported_from_h.py

```diff
@@ -0,0 +1,17 @@
+import sqlalchemy as sa
+
+from lms.db import BASE
+
+
+class CourseGroupsExportedFromH(BASE):
+    """
+    A read-only table holding a one-time export of groups from h.
+
+    See: https://github.com/hypothesis/lms/issues/1787
+
+    """
+
+    __tablename__ = "course_groups_exported_from_h"
```

Comment:
Love this ugly name! Should remind us to tidy up one day. Or will it just become part of the furniture. Who knows!